chains/kalychain/logo.svg

```diff
@@ -0,0 +1,3 @@
+  </svg>
```

Comment:
this seems to be just a png inside svg tags, do you have an actual svg that can be used instead?

Reply:
applied your suggested change